.gitignore

```diff
@@ -11,3 +11,5 @@ parts/
 doc/_build
 htmlcov/
 .coverage
+coverage.xml
+nosetests.xml
```

Comment:
Is the last line still needed. It seems to me that you dropped the nose test runner in tox.ini.